src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
Also, if you have that new plotting package UnicodePlots.jl you can add plotting to docstrings which was quite surprising:

```jl
using Docile

addhook(Docile.Docs.directives)

"""
@{example:
using UnicodePlots
lineplot(sin, 1:.5:10, color = :green)
}
"""
f(x) = x
```